blog/2024-11-2-app-platform-v12.md

```diff
@@ -0,0 +1,58 @@
+---
+slug: 2024/11/app-platform-v12
+title: App Platform v12
+authors: core
+tags: [app platform, developer tools, webapp, announcement]
+---
+
+# Platform v12 Migration Guide: Vite & React 18
+
+Good news! Vite and React 18 in the app platform are ready to use!
```

Comment:
```suggestion
Good news! With the App Platform v12 release, we've replaced the deprecated Create React App (CRA) with Vite, and upgraded React from v16 to v18!
```

Reply:
I edited this a little bit: https://github.com/dhis2/developer-portal/pull/167/commits/ff2cb6fb5d4f181b6733b08451ad71d9b0b18ef7 -- let me know what you think

Reply:
looks good!